docs/conf.py

```diff
@@ -61,10 +62,11 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
+import lasagne
 # The short X.Y version.
-version = '0.1'
+version = '.'.join(lasagne.__version__.split('.', 3)[:2]).rstrip('dev')
 # The full version, including alpha/beta/rc tags.
-release = '0.1'
+release = lasagne.__version__
```

Comment:
This line requires "import lasagne" beforehand, maybe this is not universally fine. I also don't know exactly how Sphinx works. We can just merge and see what happens.